### PR TITLE
locale-util: prefix special glyph enum values with SPECIAL_GLYPH_

### PR DESCRIPTION
**Pull request by poettering**
This has been irritating me for quite a while: let's prefix these enum
values with a common prefix, like we do for almost all other enums.

No change in behaviour, just some renaming.

### CODE REVIEW

File: src/basic/locale-util.c [hunk 1]
```diff
@@ -378,50 +378,50 @@ const char *special_glyph(SpecialGlyph code) {
...
+                        [SPECIAL_GLYPH_MU]                      = "u",
+                        [SPECIAL_GLYPH_CHECK_MARK]              = "+",
+                        [SPECIAL_GLYPH_CROSS_MARK]              = "-",
+                        [SPECIAL_GLYPH_ECSTATIC_SMILEY]         = ":-]",
+                        [SPECIAL_GLYPH_HAPPY_SMILEY]            = ":-}",
+                        [SPECIAL_GLYPH_SLIGHTLY_HAPPY_SMILEY]   = ":-)",
+                        [SPECIAL_GLYPH_NEUTRAL_SMILEY]          = ":-|",
+                        [SPECIAL_GLYPH_SLIGHTLY_UNHAPPY_SMILEY] = ":-(",
+                        [SPECIAL_GLYPH_UNHAPPY_SMILEY]          = ":-{️",
+                        [SPECIAL_GLYPH_DEPRESSED_SMILEY]        = ":-[SPECIAL_GLYPH_",
```
Comment (filbranden) [inline src/basic/locale-util.c]: something went wrong with this one...

Comment (poettering) [reply src/basic/locale-util.c]: ouch!

**Comment by poettering** [comment]
Force pushed a new version, only change is the search/replace fuckup  @filbranden noticed.

**Comment by poettering** [comment]
Setting green label, since @filbranden liked it, and the fix is so trivial.

**Pull request merged**